src/Logic.hs

```diff
@@ -407,23 +418,40 @@ handleCommentAdded triggerConfig projectConfig prId author body state =
     -- a user has push access requires an API call.
     Just pr -> do
       let approvalType = parseMergeCommand triggerConfig body
-      isApproved <- if isJust approvalType
-        then isReviewer author
-        else pure False
+      isApproved <-
+        if isJust approvalType
+          then isReviewer author
+          else pure False
+      day <- dayOfWeek . utctDay <$> getDateTime
```

Comment:
This gets the date in the UTC time zone. It will be Friday two hours later in Europe/Amsterdam than in UTC when DST is in effect. Is that intentional? (And also, does it make sense to pick midnight as the boundary? Is Saturday 04:00 an okay time to merge things?)

Reply:
Thanks for the feedback. What would be your proposal for updating the nudge? Should we detect whether it is Friday [Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)? (Start at UTC+14 / End at UTC-12)
Using AoE as the dedicated time range would already disallow the merge to proceed on Thursday afternoon in Europe/Amsterdam when DST is in effect. Which might be a valid moment for the stop merging nudge.

Reply:
Good point about the timezone! I think just detecting Friday at our location is good enough (maybe the timezone could be made configurable, but that's about it).

> Is Saturday 04:00 an okay time to merge things

It is not. But the feature is intended as a guardrail against accidents, not a foolproof method of preventing all untimely merges, and I'd just assume that there won't be accidental merges on Saturday at 04:00.

We might want to just cover the whole weekend, but I think nighttime can be ignored for the moment.

Reply:
> What would be your proposal for updating the nudge?

If it’s just a nudge on Fridays for people working in Europe/Amsterdam, I would just ignore the issue and keep it like this.

At the expense of scope creep, maybe a nicer solution that also covers the weekend is to provide a time range in the configuration file. It could include a start day of week + time of day and an end day of week + time of day, and a time zone in which to interpret them.